### ISSUE TITLE
Real Solar System support

### ISSUE THREAD

**Issue by alexanderabramov** [alexmoon/ksp#20]
Could you add an option for real solar system? I guess anyone can add custom bodies in the calculator as a workaround, but they won't be saved for the future.
I see it as a simple dropdown Kerbol/Sol that loads pre-configured parameters.

I also wonder if there is such a calculator online for actual real solar system, could not find one with a cursory search. If you know of one maybe add a link to the page?

**Comment by mark-c** [comment]
Yeah, cookies (if that's possible) would be really nice. I have had to enter my vessel orbit data already three times, with no success, because the destination box turns off.
